docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+    };
+
+    const gasConfigB = {
+        maxFeePerGas: ethers.BigNumber.from("1500000000"),
+        chainId: LIT_CHAINS[chainBParams.chain].chainId,
+        nonce: await chainBProvider.getTransactionCount(mintedPKP.ethAddress),
+    };
+
+    await litNodeClient.connect();
+
+    const results = await litNodeClient.executeJs({
+        ipfsId: action_ipfs,
+        sessionSigs: sessionSigs,
+        jsParams: {
+            pkpPublicKey: mintedPKP.publicKey,
+            pkpAddress: mintedPKP.ethAddress,
+            authSig: JSON.stringify(authSig),
+            chainAGasConfig: gasConfigA,
+            chainBGasConfig: gasConfigB,
+        },
+    });
+
+    console.log("results: ", results);
+```
+
+Based on the results returned by the nodes, we can execute the signed transactions by sending them to the blockchain networks.
+
+```js
+
+    if (results.signatures == undefined) {
+        return;
+    }
+
+    else if (results.signatures.chainBSignature == undefined) {
+        console.log("executing clawbackA tx..")
+        await executeTxA(results, chainAProvider);
+    }
+
+    else if (results.signatures.chainASignature == undefined) {
+        console.log("executing clawbackB tx..")
+        await executeTxB(results, chainBProvider);
+    }
+
+    else {
+        console.log("executing swap txs..")
+        await executeTxA(results, chainAProvider);
+        await executeTxB(results, chainBProvider);
+    }
+}
+```
+
+These two functions will handle sending individual transactions on each network.
+
+```js
+
+async function executeTxA(results, chainAProvider) {
+    const signatureA = formatSignature(results.signatures.chainASignature);
+    const tx1 = await chainAProvider.sendTransaction(
+        ethers.utils.serializeTransaction(
+            results.response.chainATransaction,
+            signatureA
+        )
+    );
+    console.log(tx1);
+    
+    const receipt1 = await tx1.wait();
+    const blockExplorer1 = LIT_CHAINS[chainAParams.chain].blockExplorerUrls[0];
+    
+    console.log(`tx: ${blockExplorer1}/tx/${receipt1.transactionHash}`);
+}
+
+async function executeTxB(results, chainBProvider) {
+    const signatureB = formatSignature(results.signatures.chainBSignature);
+    const tx2 = await chainBProvider.sendTransaction(
+        ethers.utils.serializeTransaction(
+            results.response.chainBTransaction,
+            signatureB
+        )
+    );
+    const receipt2 = await tx2.wait();
+    const blockExplorer2 = LIT_CHAINS[chainBParams.chain].blockExplorerUrls[0];
+
+    console.log(`tx: ${blockExplorer2}/tx/${receipt2.transactionHash}`);
+}
+```
+
+A signed transaction returned by the Lit Network has a different structure for signatures that we need to reconstruct as follows:
+
+```js
+function formatSignature(signature) {
+    const encodedSig = ethers.utils.joinSignature({
+        v: signature.recid,
+        r: `0x${signature.r}`,
+        s: `0x${signature.s}`,
+    });
+    return encodedSig;
+}
+```
```

Comment:
I think it could be worth including a "Summary" section which recaps all of the above steps and defines success criteria

Reply:
https://github.com/LIT-Protocol/docs/pull/330/commits/30185e8fdd79532766306cf2736f33c5726e9f50#diff-0e36021eb2e9b13aa34e50ec77474d75d50bab8d18cba35c954841b81d9efee7R656